# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/binutils/hotfix-gold-sparcv.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- binutils-2.39/gold/configure.tgt.vanilla	2022-10-31 19:11:38.990566812 +0100
+++ binutils-2.39/gold/configure.tgt	2022-10-31 19:11:51.724565487 +0100
@@ -80,7 +80,7 @@
   targ_big_endian=false
   targ_extra_big_endian=true
   ;;
-sparc-*)
+sparc*-*)
   targ_obj=sparc
   targ_machine=EM_SPARC
   targ_size=32
